testapp/src/main/res/values/strings.xml

```diff
@@ -21,7 +22,7 @@
     <string name="userdata_warning_empty_contact_list">No contact available!</string>
 
     <string name="action_display">Display</string>
-    <string name="action_go_input">Display Miniapp From ID</string>
+    <string name="action_go_input">Display Miniapp From ID or URL</string>
```

Comment:
It's a little confusing that this is only available when "Preview mode" is turned off. Could we also display this button while the Demo App has "Preview mode" enabled but then change the text to `Display Miniapp from URL` and hide the "App ID" field and button on `mini_app_input_activity.xml`?

This would be a little more complex to do. What do you think?

@minh-rakuten @khairul-alam-licon Any other suggestions for how to handle this?

Reply:
if loading a MiniApp by url has no dependency with preview/published mode, then we can use a separate place for this option. I am not good at suggesting UX design, but the simplest thing for now I can say is adding a separate button on bottom of this UI.

Reply:
Yeah I intended that we will go that way in separated PR. It's okay to implement it in this PR too.
I think the label can be changed into `Display Miniapp by Input`, it would go well with both cases.

Reply:
I think I'll go with @corycaywood 's solution. Should be fairly easy to implement